Modules/TPC/include/TPC/Utility.h

```diff
@@ -40,6 +40,17 @@ void addAndPublish(std::shared_ptr<o2::quality_control::core::ObjectsManager> ob
 /// \return std::vector<TCanvas*>
 std::vector<TCanvas*> toVector(std::vector<std::unique_ptr<TCanvas>>& input);
 
+/// \brief Fills std::vector<std::unique_ptr<TCanvas>> with data from calDet
+/// \param calDet Object to be displayed in the canvases
+/// \param canvases Vector containing three std::unique_ptr<TCanvas>, will be filled
+/// \param params Information about the ranges of the histograms that will be drawn on the canvases
+/// \param paramName Name of the observable that is stored in calDet
+void fillCanvases(const o2::tpc::CalDet<float>& calDet, std::vector<std::unique_ptr<TCanvas>>& canvases, std::unordered_map<std::string, std::string>& params, const std::string paramName);
```

Comment:
```suggestion
void fillCanvases(const o2::tpc::CalDet<float>& calDet, std::vector<std::unique_ptr<TCanvas>>& canvases, const std::unordered_map<std::string, std::string>& params, const std::string paramName);
```

Reply:
I tried this in the first place but it leads to
````
error: passing 'const std::unordered_map<std::__cxx11::basic_string<char>, std::__cxx11::b
asic_string<char> >' as 'this' argument discards qualifiers [-fpermissive]
   62 |                                         std::stoi(params[(paramName + "NBins")]), std::stof(params[(paramName + "XMin")]), std::stof(params[(paramName + "XMax")]),
      |                                                                               ^
````

Reply:
Ok, fine.

Reply:
I will try something like this. A check for missing parameters makes sense anyway.